stagecraft/apps/collectors/migrations/0003_auto_20150730_1329.py

```diff
@@ -0,0 +1,156 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
```

Comment:
If we do this as a data migration, can we re run it if needed? I thought in the past once we had run the migration once, django wouldnt allow re running it, which can be quite annoying if there is a mistake